addons/website_slides/models/slide_slide.py

```diff
@@ -616,7 +616,7 @@ def create(self, vals_list):
                     # only update keys that are not set in the incoming vals
                     slide.update({key: value for key, value in slide_metadata.items() if key not in vals.keys()})
 
-            if not 'completion_time' not in vals:
+            if 'completion_time' not in vals:
```

Comment:
Hello @std-odoo 


Did you mean JS test case or JS test tour.
Because problem is coming from the front end not backend
and there is no JS Test in this module so do you want me write JS test tour.

Thanks :slightly_smiling_face:

Reply:
Hello @std-odoo 

Oh Sorry My bad :sweat_smile:
Was thinking about something else.

Thanks :slightly_smiling_face: